### PR TITLE
docs(analytics): add screen_class to navigation screen tracking examples

### PR DESCRIPTION
**Pull request by MohamadKh75**
### Related issues

https://github.com/invertase/react-native-firebase/pull/4145

### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [x] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

File: docs/analytics/screen-tracking.md [hunk 1]
```diff
@@ -50,7 +51,7 @@ Navigation.events().registerComponentDidAppearListener(async ({ componentName, c
   if (componentType === 'Component') {
     await analytics().logScreenView({
       screen_name: componentName,
-      screen_class: componentName,
+      screen_class: componentName
```
Comment (mikehardy) [inline docs/analytics/screen-tracking.md:54]: this is a matter of taste, with the community (and the default / our default prettier config) leaning towards trailing commas these days: https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker

```suggestion
      screen_class: componentName,
```

File: docs/analytics/screen-tracking.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ import { NavigationContainer } from '@react-navigation/native';
     if (previousRouteName !== currentRouteName) {
       await analytics().logScreenView({
         screen_name: currentRouteName,
+        screen_class: currentRouteName
```
Comment (mikehardy) [inline docs/analytics/screen-tracking.md:32]: Same as the other one - we default to trailing commas now. Super tiny nitpick, I know

```suggestion
        screen_class: currentRouteName,
```

**Comment by mikehardy** [comment]
happened to be scanning the queue in between tasks and saw it right as it came in ;-)
Thanks for this!

**Comment by MohamadKh75** [comment]
Thanks @mikehardy
I really appreciate your efforts :heart:

**Pull request merged**